rust/sdk-processor/src/config/indexer_processor_config.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use super::processor_config::ProcessorConfig;
+use crate::processors::events_processor::EventsProcessor;
+use anyhow::Result;
+use aptos_indexer_processor_sdk::aptos_indexer_transaction_stream::TransactionStreamConfig;
+use sdk_server_framework::RunnableConfig;
+use serde::{Deserialize, Serialize};
+
+pub const QUERY_DEFAULT_RETRIES: u32 = 5;
+pub const QUERY_DEFAULT_RETRY_DELAY_MS: u64 = 500;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct IndexerProcessorConfig {
+    pub processor_config: ProcessorConfig,
```

Comment:
not necessary in this PR, but I think we should consider supporting running multiple processors here.

Reply:
Can you explain what you mean and why? If you want to extract different types of data using the same TransactionStream, you could use the fanout method in ProcessorBuilder. But I think we shouldn't support multiple processors in the config, that fundamentally changes how we do deployment

Reply:
What I'm saying is we should provide the `fanout to multiple processors after receiving the grpc stream` functionality as part of the sdk/library. We should make it available at config level so it's much easier to use, otherwise if it requires code change, it will be less likely for people to adopt to it.

Reply:
Got it. This sounds like a separate feature built on top of SDK which will require SDK + config + deployment changes, almost like config as code which @banool  and @just-in-chang have sort of been looking into for the low code indexing.

Reply:
and for our own deployment, if you look at the profiling result, the stream part is taking a significant portion of CPU for most of the processors, and it takes a lot of network bandwidth, too.

Reply:
> This sounds like a separate feature built on top of SDK which will require SDK + config + deployment changes

I think it should be on top of `aptos_indexer_processor_sdk`, but as part of `rust/sdk-processor`.

Reply:
> but as part of rust/sdk-processor

I don't necessarily agree with this. This has implications on processor performance, like if 1 processor is behind or stuck all of them will be behind.

Reply:
I understand there will be some implications, what I'm saying is we should provide the options, and allow the user to choose whichever they think better. There were people explicitly asking for this, and I do see the pros would outweigh the cons in many cases.

Reply:
It can be built with the components offered by SDK. We can provide an example of sharing 1 stream with multiple "processors" once we work on the documentation.